config/application.example.yml

```diff
@@ -0,0 +1,64 @@
+# Add application configuration variables here, as shown below.
```

Comment:
This is exactly the same file as `config/application.yml.example`. Can't we use the existing file? Or rename it to this one if we need to? That would just need some other changes in other places.

Reply:
Right, forgot to delete the old one, just updated it.